dashboard/app/models/foorm/library_question.rb

```diff
@@ -17,46 +17,10 @@
 #
 
 class Foorm::LibraryQuestion < ApplicationRecord
-  include Seeded
```

Comment:
I don't understand if I should keep this here, since this model is still technically seeded (albeit by LibraryQuestion now). I don't see where the methods provided by `Seeded` are used, but I could be missing something.

Reply:
i don't know anything about this, but would be good to understand it before merging.

Reply:
I'll just leave it since the model is still seeded, it basically just provides some [utility methods](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/concerns/seeded.rb) for seeded models.